Symfony/CS/ConfigInterface.php

```diff
@@ -72,6 +72,20 @@ public function setDir($dir);
     public function getDir();
 
     /**
+     * Sets the directory into which the cache file would be written if caching were enabled.
+     *
+     * @return string
+     */
+    public function setCacheDir($dir);
```

Comment:
@keradus 

To be honest, I was thinking the same, just wasn't sure. Shall I amend?